spring-cloud-config-client/src/main/java/org/springframework/cloud/config/client/ConfigServicePropertySourceLocator.java

```diff
@@ -69,14 +69,20 @@
 
 	private ConfigClientProperties defaultProperties;
 
+	private final String ACTIVE_PROFILES_PROPERTY_NAME = "spring.profiles.active";
+
 	public ConfigServicePropertySourceLocator(ConfigClientProperties defaultProperties) {
 		this.defaultProperties = defaultProperties;
 	}
 
 	@Override
-	@Retryable(interceptor = "configServerRetryInterceptor")
```

Comment:
I think it needs to stay here for some rare backwards compatibility issues since the call to `locate(properties)` wouldn't be retryable.